.cargo/config_fast_builds.toml

```diff
@@ -83,12 +83,19 @@ rustflags = [
   # - Ubuntu: `sudo apt-get install mold clang`
   # - Fedora: `sudo dnf install mold clang`
   # - Arch: `sudo pacman -S mold clang`
-  # "-Clink-arg=-fuse-ld=/usr/bin/mold",
```

Comment:
Good catch! When I wrote this I didn't realize `/usr/bin` was automatically in the `PATH`.